cmd/create/cluster/cmd.go

```diff
@@ -2926,6 +2924,24 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 	return command
 }
 
+func buildTagsCommand(tags map[string]string) []string {
+	// set correct delim, if a key or value contains `:` the delim should be " "
+	delim := ":"
```

Comment:
NP
Do we need to calculate the delimiter again? or do we have it from the previous check `delim := aws.GetTagsDelimiter(sanitizedTags)`

Reply:
Different types, `GetTagsDelimiter` is getting the delim from an array of strings, when building the tags command we are using map[string]string, so we are calculating based off the key value pairs :thinking:

Reply:
given the number of params `buildCommand` has, I wouldn't like to pass the delim down from a previous calc :thinking:  
maybe a tech debt ticket to improve the `buildCommand` and remove all those params we are passing down and we can improve there.